django_singlestore/features.py

```diff
@@ -318,9 +318,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_non_deterministic_collations = True
 
     # Does the backend support column and table comments?
-    supports_comments = True
+    supports_comments = False
     # Does the backend support column comments in ADD COLUMN statements?
-    supports_comments_inline = True
+    supports_comments_inline = False
```

Comment:
I think we should keep these as True as SingleStore supports comments. We should skip the failing tests and add a note that the comments cannot be altered to the README (where we talk about linmitations).